pinot-query-runtime/src/main/java/org/apache/pinot/query/service/QueryConfig.java

```diff
@@ -22,9 +22,11 @@
  * Configuration for setting up query runtime.
  */
 public class QueryConfig {
-  public static final int DEFAULT_MAX_INBOUND_QUERY_DATA_BLOCK_BYTES_SIZE = 128 * 1024 * 1024;
   public static final long DEFAULT_TIMEOUT_NANO = 10_000_000_000L;
 
+  public static final String KEY_OF_MAX_INBOUND_QUERY_DATA_BLOCK_BYTES_SIZE = "pinot.query.runner.max.msg.size";
```

Comment:
Shall we use "multistage" in the name somewhere to clearly distinguish between multistage engine specific configs vs rest ?

Reply:
you mean the config key value or the static variable name? 
- the variable is already within the multistage `QueryConfig` class so it should be good. 
- I use `pinot.query.runner` prefix to differentiate between normal query and multi-stage query runner, but yeah I am open to other key naming convension

Reply:
I meant the config key value. Was thinking something like `pinot.query.mulistage`

Reply:
If you agree, we can do this as follow up cc @walterddr

Reply:
Yeah it is still experimental so yes we can use different key prefix by changing it systematically